xapian-letor/.gitignore

```diff
@@ -8,10 +8,14 @@ autom4te.cache
 bin/.deps
 bin/.dirstamp
 bin/.libs
-bin/questletor
-bin/questletor.1
+bin/xapian-rank
+bin/xapian-rank.1
 bin/xapian-letor-update
 bin/xapian-letor-update.1
+bin/xapian-prepare-trainingfile
+bin/xapian-prepare-trainingfile.1
+bin/xapian-train
+bin/xapian-train.1
 config.guess
```

Comment:
When there's a sorted list and you add or modify an entry, it's helpful to preserved the sorted nature (so `xapian-rank` would be just before `xapian-train`).